src/components/AiMarketplace/ServiceListingHeader/index.js

```diff
@@ -0,0 +1,97 @@
+import React, { Component, Fragment } from "react";
+import { connect } from "react-redux";
+import Slider from "react-slick";
+import "slick-carousel/slick/slick.css";
+import "slick-carousel/slick/slick-theme.css";
+
+import Grid from "@material-ui/core/Grid";
+import { withStyles } from "@material-ui/styles";
+import StarRateIcon from "@material-ui/icons/StarRate";
+
+import { useStyles } from "./styles";
+import StyledButton from "../../common/StyledButton";
+import { uiContentActions } from "../../../Redux/actionCreators";
+
+class ServiceListingHeader extends Component {
+  componentDidMount = () => {
+    const { fetchCarousel } = this.props;
+    fetchCarousel();
+  };
+
+  render() {
+    const { classes, carousel } = this.props;
+
+    const settings = {
+      infinite: false,
+      speed: 500,
+      slidesToShow: 1,
+      slidesToScroll: 1,
+    };
+
+    return (
+      <Fragment>
+        {carousel.length > 0 ? (
```

Comment:
use guard clause instead of ternary operator. 

e.g
if(!carousel.lenght>0){return null}
return(....)